frontend/test/jest-setup.js

```diff
@@ -1,4 +1,3 @@
-import { XMLHttpRequest } from "xmlhttprequest";
```

Comment:
We added this when introducing `nock` in https://github.com/metabase/metabase/pull/27337. It seems we don't really need it.